comparisontool/tests/settings.py

```diff
@@ -0,0 +1,46 @@
+import dj_database_url
```

Comment:
Having this in the settings means that unless a developer installs the testing dependencies with

```
pip install -e '.[testing]'
```

then they'll get an `ImportError` here:

```bash
$ mkvirtualenv dccc
(dccc) $ pip install -e .
(dccc) $ ./manage.py migrate
$ ./manage.py migrate
Traceback (most recent call last):
...
    import dj_database_url
ImportError: No module named dj_database_url
```

I see two options:

1. We could remove [this](https://github.com/cfpb/django-college-costs-comparison/blob/master/setup.py#L45)   `extras_require` section in setup.py and move `dj-database-url` as a dependency to `install_requires`. This would simplify but would have the downside of requiring this as a dependency even if you don't want to use that functionality.
2. We could move this import down with the environment variable check, making it totally optional:

    ```py
    try:
        import dj_database_url
    except ImportError:
        # dj-database-url is an optional dependency to support use of DATABASE_URL.
        pass
    else:
        if 'DATABASE_URL' in os.environ:
            DATABASES['default']  =  dj_database_url.config()
    ```

I have a weak preference for the first option for simplicity's sake; the second option feels more hidden as we'd have to document the need to do an additional `pip install dj-database-url` step if you want to use this environment variable. Additionally having the dependency as part of the default installation doesn't feel terribly heavy to me as it's such a lightweight package.

Reply:
I agree: I also have a weak preference for option 1. I've implemented that in 5f6148ab11736dae6762a2ac7c092762dc25b9a2.